src/design-tokens/tier-1-definitions/typography-presets.css

```diff
@@ -120,13 +120,27 @@
 @define-mixin eds-typography-preset-9 $important {
   font-family: var(--eds-font-family-tertiary) $important;
   font-weight: var(--eds-font-weight-regular) $important;
-  font-size: var(--eds-font-size-md-2) $important;
-  line-height: var(--eds-line-height-sm) $important;
-  letter-spacing: 1.5px $important;
-  text-transform: uppercase $important;
+  font-size: var(--eds-font-size-sm) $important;
+  /* TODO: add this to line height variables */
+  line-height: var(--eds-size-2-and-half) $important;
+  letter-spacing: normal $important;
 }
 
 @define-mixin eds-typography-preset-9-bold $important {
   @mixin eds-typography-preset-9 $important;
   font-weight: var(--eds-font-weight-bold) $important;
 }
+
+@define-mixin eds-typography-preset-10 $important {
+  font-family: var(--eds-font-family-tertiary) $important;
+  font-weight: var(--eds-font-weight-regular) $important;
+  font-size: var(--eds-font-size-xs) $important;
+  /* TODO: add this to line height variables */
+  line-height: var(--eds-size-1-and-half) $important;
+  letter-spacing: normal $important;
+}
+
+@define-mixin eds-typography-preset-10-bold $important {
+  @mixin eds-typography-preset-10 $important;
+  font-weight: var(--eds-font-weight-bold) $important;
+}
```

Comment:
`preset-10` is the new medium + small button styles (the only difference between them is padding).